packages/e2e/tests/login.test.ts

```diff
@@ -0,0 +1,62 @@
+import { expect, test } from '@playwright/test';
+
+test('should redirect to login when not logged in', async ({ page }) => {
+  await page.goto('/');
+  await expect(page).toHaveURL('/login');
+});
+
+test('should reject unknown credentials', async ({ page }) => {
+  await page.goto('/login');
+  await page.getByLabel('Email').fill('user@votura.org');
+  await page.getByLabel('Password').fill('1234');
+  await page.getByRole('button', { name: 'Login' }).click();
+
+  await expect(page.getByText('Could not login')).toBeVisible();
+  await expect(page.getByText('Could not login')).toBeVisible();
+  await expect(page).toHaveURL('/login');
+});
+
+test('should reject invalid email format', async ({ page }) => {
+  await page.goto('/login');
+  await page.getByLabel('Email').fill('foo');
+  await page.getByRole('button', { name: 'Login' }).click();
+
+  await expect(page.getByText('Invalid email address.')).toBeVisible();
+});
```

Comment:
Please add URL check
```suggestion
test('should reject invalid email format', async ({ page }) => {
  await page.goto('/login');
  await page.getByLabel('Email').fill('foo');
  await page.getByRole('button', { name: 'Login' }).click();

  await expect(page.getByText('Invalid email address.')).toBeVisible();
  await expect(page).toHaveURL('/login');
});
```